server/build.gradle

```diff
@@ -334,7 +334,22 @@ if (isEclipse == false || project.path == ":server-tests") {
     classpath = project.test.classpath
     testClassesDirs = project.test.testClassesDirs
     include '**/*IT.class'
+    // TODO: remove ScriptedMetricAggParamsV6CompatIT for 7.0
+    exclude '**/ScriptedMetricAggParamsV6CompatIT.class'
   }
+
+  // TODO: remove this task dedicated to ScriptedMetricAggParamsV6CompatIT for 7.0
+  task scriptedMetricAggDeprecatedParamIntegTest(type: RandomizedTestingTask,
```

Comment:
I considered centralizing this similar to the `additionalTest` extension for unit tests but opted not to at this point since it seems like overkill with no other known uses for it. Even `additionalTest` only appears to be used in two spots right now.

Reply:
I don't think we need an IT. The purpose of the test only needs to be that the sysprop is read appropriately. The behavior can and should all be tested in a unit test.